docs/reference/files.md

```diff
@@ -80,7 +80,7 @@ exist, it is dynamically generated and immediately returned.
 
 ### Preset Transformations
 
-- **`key`** — This **key** of the [Storage Asset Preset](/user-guide/cloud/project-settings#files-thumbnails), a
+- **`key`** — This **key** of the [Storage Asset Preset](/packages/@directus/types/type-aliases/type-alias.SettingsStorageAssetPreset.html), a
```

Comment:
Thank you @ChuckMoe! I'd rather keep linking to a non-technical explanation of transformation presets:

```suggestion
- **`key`** — This **key** of the [Storage Asset Preset](/user-guide/settings/project-settings#files-storage), a
```